agent/util-scripts/pbench-register-tool

```diff
@@ -232,6 +232,7 @@ debug_log "tool_opts: \"${tool_opts[@]}\""
 
 # Determine what our local hostname(s) and IP(s) are.
 # FIXME: this does not check for ipv6
+if ! command -v ip >/dev/null 2>&1 ; then echo "PATH=$PATH"; fi
```

Comment:
This is, strictly speaking, debugging code, but it was a little hard to set up (believe it or not), and it was so very illuminating when it triggered, that I'm inclined to leave it in place:  the code below it depends on having `/usr/sbin` in the User's `PATH` in order to find the `ip` command, and, if it is not there (as it is not, by default, in Mr. Jenkins's environment), it comes as a nasty surprise, which the output of this command will help diminish.  (And, otherwise, it has no effect.)

Reply:
Why not move this to the beginning of `jenkins/run-unittests` so that we don't have to modify exiting code already being shipped?

Reply:
The crux of the matter is, if the user's environment is not set up correctly, then things will go off the rails at the very next statement, regardless of how this command is invoked, and so this is the appropriate point at which to detect, diagnose, and/or report the failure.  However, this is rather cryptic way to do it, and, it would be best to prevent this problem rather than to diagnose it on the fly, so, I'll revert this change so that this PR doesn't impact any of the product code.

See also #2383.